packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -433,8 +442,8 @@ export class ConnectionService {
       )
     }
 
-    connectionRecord.errorMsg = `${connectionProblemReportMessage.description.code} : ${connectionProblemReportMessage.description.en}`
-    await this.updateState(connectionRecord, ConnectionState.None)
+    connectionRecord.errorMessage = `${connectionProblemReportMessage.description.code} : ${connectionProblemReportMessage.description.en}`
+    await this.updateState(connectionRecord, connectionRecord.state)
```

Comment:
The state won't change, so I think we should just call the connection repository with update directly as this will emit a state changed event.

Reply:
Also, with the `None` state being removed, there is no way to determine that you received a problem report, so we should probably look into adding some events related to problem reports in follow up PRs

Reply:
Hi @TimoGlastra, If we update the `updateState` with an `update`, it will be difficult to capture the event in the e2e test case?

https://github.com/hyperledger/aries-framework-javascript/blob/1041aa77bdc1c441d99e7852d4062766520d6597/packages/core/tests/credentials.test.ts#L483